app/views/users/show.html.erb

```diff
@@ -12,10 +12,20 @@
 
 </section>
 
+<% if current_user.skills.size > 0 %>
 <section class="my-skills">
-
+  <h3>My Skills:</h3>
+  <ul>
+    <% current_user.skills.each do |skill| %>
+    <li><%= skill.skill %></li>
+    <% end %>
+  </ul>
 </section>
-
+<% else %>
+ <section class='my-skills'>
+  <p>You currently have no skills</p>
```

Comment:
can we change this to "You have not added any skills." to be more polite?

Reply:
Sure

Reply:
Nice catch @twarbelow